kobo/settings/base.py

```diff
@@ -42,6 +42,12 @@
     SESSION_COOKIE_SECURE = True
     CSRF_COOKIE_SECURE = True
 
+if os.environ.get("SECURE_HSTS_INCLUDE_SUBDOMAINS", "False") == "True":
+    SECURE_HSTS_INCLUDE_SUBDOMAINS = True
+if os.environ.get("SECURE_HSTS_PRELOAD", "False") == "True":
+    SECURE_HSTS_PRELOAD = True
+SECURE_HSTS_SECONDS = os.environ.get("SECURE_HSTS_SECONDS", 0)
+
```

Comment:
FYI: We tend to use single quotes for strings, at least for new code :-) 
We keep double quotes for triple double quotes or f-strings containing single quotes.

Reply:
Updated and rebased on latest beta